src/components/not-found.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { NimbleEmoji } from '.'
+
+export default class NotFound extends React.PureComponent {
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    var { data, emojiProps, i18n, notFound } = this.props
+
+    const imageComponent =
+      !notFound() ? (
```

Comment:
Calling `notFound` here would create a React component. If we're doing that, we should probably save the result. 

You can rewrite this component as the following, to save creating the component just to check if `notFound` is defined:

```js
export default class NotFound extends React.PureComponent {
  render() {
    const { data, emojiProps, i18n, notFound } = this.props

    const imageComponent = (notFound && notFound()) || (
      <div>
        {NimbleEmoji({
          data: data,
          ...emojiProps,
          size: 38,
          emoji: 'sleuth_or_spy',
          onOver: null,
          onLeave: null,
          onClick: null,
        })}
        <div className="emoji-mart-no-results-label">{i18n.notfound}</div>
      </div>
    );

    return (
      <div className="emoji-mart-no-results">
        {imageComponent}
      </div>
    )
  }
}
```